src/Domain/CoordinateMaps/CoordinateMap.tpp

```diff
@@ -0,0 +1,417 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+
+#include <algorithm>
+#include <array>
+#include <boost/optional.hpp>
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+#include <pup.h>
+#include <tuple>
+#include <type_traits>
+#include <typeinfo>
+#include <unordered_map>
+#include <vector>
+
+#include "DataStructures/Tensor/Identity.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Parallel/PupStlCpp11.hpp"
+#include "Utilities/ForceInline.hpp"
```

Comment:
don't need this

Reply:
Also fix the includes in the hpp file (I think you can remove array, algorithm cstdint, typeinfo, Identity, ForceInline, Gsl, MakeArray, Tuple)